facebookresearch/ParlAI
# Add file for helper metrics for slots

SlotF1Metric + SlotF1Metric test imported from an internal implementation. Will go through and refactor + verify that I don't break existing tasks using this next. 

**Testing steps**
`pytest tests/SlotF1Metric + SlotF1Metric test imported from an internal implementation. Will go through and refactor + verify that I don't break existing tasks using this next. 

**Testing steps**
`pytest tests/test_slot_metrics.py`

Note that I did change the SlotF1Metric class (and its associated test) a bit in the 2nd change in this PR in order to get tests to run properly. (I got assertion errors trying to run the `pytest test_slot_f1_metrics.py` in its original internal directory so figured it would need to change anyhow.)
